GoogleAnalyticsTracker.Core/TrackerBase.cs

```diff
@@ -26,7 +26,11 @@ public partial class TrackerBase : IDisposable
         public bool ThrowOnErrors { get; set; }        
         public string EndpointUrl { get; set; }
 
-        /// <summary> Use HTTP GET (not recommended) instead of POST.</summary>
+        /// <summary> 
+        /// Use HTTP GET (not recommended) instead of POST.
+        /// When switched on, the the <see cref="AmendParameters(TrackerParameters.Interface.IGeneralParameters)"/>
+        /// sets the Sets the <see cref="IGeneralParameters.CacheBuster"/>, too.
```

Comment:
Funky comment :-) (sets the sets the)

Reply:
"the the" too. Stammering comment. I'll fix that, thanks.